lesson11/main.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const body = document.getElementById("body");
+const wrapper = document.createElement("div");
+body.appendChild(wrapper);
+const startLoading = () => {
+   const loadingIcon = document.createElement("img");
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+const endLoading = () => wrapper.remove();
+const jsonURL = "https://jsondata.okiba.me/v1/json/KKl45211103113313";
+
+async function fetchData() {
+   startLoading();
+   try {
+      const response = await fetch(jsonURL);
+      const json = await response.json();
+      return json.data;
+   } catch(error) {
+      alert('データの取得に失敗しました。')
+      console.error('エラー');
+   } finally {
+      console.log('処理が終了しました。')
+      endLoading();
+   }
+}
+
+async function runCreateListElement() {
+   try {
+      const result = await fetchData();
+      createListElements(result);
+   } catch {
+      alert(`データが取得できませんでした`)
+      console.error('エラー');
+   }
+};
+
+const createListElements = (data) => {
+   const fragment = document.createDocumentFragment();
+   const lists = document.getElementById("lists");
+   for (let i = 0; i < data.length; i++){
+      const li = document.createElement("li");
+      const anchor = document.createElement("a");
+      const image = document.createElement("img");
+      const text = document.createTextNode(data[i].text);
+      anchor.href = `/${data[i].a}` + `.html`;
```

Comment:
I think you don't have to use '+' and separate them.
like this↓
`/${data[i].a}.html`

Reply:
Thank you for your review.
I changed the part you pointed out.